noir-projects/aztec-nr/aztec/src/test/helpers/cheatcodes.nr

```diff
@@ -161,9 +166,13 @@ unconstrained fn oracle_get_side_effects_counter() -> u32 {}
 #[oracle(advanceBlocksBy)]
 unconstrained fn oracle_advance_blocks_by(blocks: u32) {}
 
+#[oracle(advanceTimestampBy)]
+unconstrained fn oracle_advance_timestamp_by(duration: u64) {}
+
 #[oracle(getPrivateContextInputs)]
 unconstrained fn oracle_get_private_context_inputs(
     historical_block_number: u32,
+    historical_timestamp: u64,
```

Comment:
Maybe I'm missing something but this feels very weird to me. Shouldn't this be an either/or situation? Context has to be requested for a block or a timestamp, providing both seems weird to me

Reply:
Lovely, thank you!